docs/products/kafka/kafka-mirrormaker/howto/log-analysis-offset-sync-tool.md

```diff
@@ -0,0 +1,65 @@
+actions. If any issues arise, [create a support ticket](/docs/platform/howto/support) or
+email [Aiven Support](mailto:support@aiven.io).
+
+| Message ID | Script message | What it means | Customer action | How to inspect | How to analyze inspection | Issue with Apache Kafka or MirrorMaker 2 |
+|------------|----------------|---------------|-------------------|----------------|----------------------------|-----------------------------------------|
+| 1 | Successfully synced, operating normally | Data and offsets are properly migrated. Consumers will start consuming from the synced offsets with minimal redelivery. | No action required | Not required | Not required | No |
+| 2 | Intentionally not synced because the source partition is empty. Other partitions in this group are synced. | The source partition has no data, and MirrorMaker does not sync these. Consumers will start consuming from the beginning when new data arrives. | No action required | Not required | Not required | No |
+| 3 | Intentionally not synced because the source partition is empty. Other partitions in this group are not synced. | The source partition has no data, and MirrorMaker does not sync these. Consumers will start consuming from the beginning when new data arrives. | No action required | Not required | Not required | No |
+| 4 | Intentionally not synced because the source group is too old. Other partitions in this group are synced. | The source partition has data, but it is newer than the committed offset for the consumer group. Consumers will start consuming from the beginning when new data arrives. | No action required | Not required | Not required | No |
+| 5 | Intentionally not synced because the source group is too old. Other partitions in this group are not synced. | The source partition has data, but it is newer than the committed offset for the consumer group. Consumers will start consuming from the beginning when new data arrives. | No action required | Not required | Not required | No |
+| 6 | Erroneously not synced. Source partition has data, and other partitions for this group are synced. | There may be an issue with offset syncing. Consumers will start consuming from the beginning when new data arrives. | [Create a support ticket](/docs/platform/howto/support) or email [Aiven Support](mailto:support@aiven.io). | Use `kafka-get-offsets` to inspect source topic or CSV output. Compare earliest and latest offsets. | Investigate potential offset syncing issues. | Yes |
+| 7 | Erroneously not synced. Source partition has data, and other partitions for this group are not synced. | There may be a systematic failure affecting all partitions in this topic. Consumers will start from the beginning when new data arrives. | [Create a support ticket](/docs/platform/howto/support) or email [Aiven Support](mailto:support@aiven.io). | Use diagnostic tools to analyze per-partition sync issues. | Investigate potential sync failure across all partitions. | Yes |
```

Comment:
Where there is an issue, we should also direct the user to raise a ticket with Aiven support

Reply:
Is this added for messages ID 6 and 7: For example, Consumers will start from the beginning when new data arrives. | [Create a support ticket](/docs/platform/howto/support) or email [Aiven Support](mailto:support@aiven.io).